src/CssInlinerPlugin.php

```diff
@@ -48,6 +52,30 @@ public function handleSymfonyEvent(MessageEvent $event): void
         $this->handleSymfonyEmail($message);
     }
 
+    private function buildCssContent(): void
+    {
+        $this->addCssFromFiles();
+        $this->addCssFromContent();
+    }
+
+    private function addCssFromFiles(): void
+    {
+        $filesToInline = $this->config['css-files'] ?? [];
+
+        $this->cssToAlwaysInclude = $this->loadCssFromFiles($filesToInline);
+    }
+
+    private function addCssFromContent(): void
+    {
+        $contentToInline = $this->config['css-content'] ?? null;
+
+        if (! $contentToInline) {
+            return;
+        }
+
```

Comment:
**Suggestion**: Insert this line
```
if ($contentToInline instanceof \Closure)  $contentToInline = $contentToInline();
````

In most cases, the css content will only be used (and this code reached) when an email is being prepared for sending, which may be no more than 1% of page loads depending on the project. So for better performance, lazy loading should be allowed.

With the line above inserted, instead of passing `Vite::content($asset)` directly, we can pass `fn() => Vite::content($asset)`.